tests/test_stlcontainers.py

```diff
@@ -656,8 +656,6 @@ def test_map_str_vector_double():
     uismap = isinstance([1.0, -65.5555, 1.0, -65.5555], Mapping) 
     m['Aha'] = [42.42, 18, 42.42, 18]
     m['Take'] = [1.0, -65.5555, 1.0, -65.5555]
-    import pprint
```

Comment:
Well, they were causing the tests to fail on Python 3 due to newly unhashable instance methods.  So it is an evolution of the language that we have to adapt around.  I also argue that printing in tests is only useful as a debugging mechanism when the test is failing and it is unclear why.  A more explicit test is better than a print, anyway. Also most of the time what is printed here is hidden anyway.  So I don't think we loose much of value and we gain a passing test suite. These tests are pretty thorough anyway.